src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheckExamplesTest.java

```diff
@@ -34,18 +34,19 @@ protected String getPackageLocation() {
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-
+            "17:5: " + getCheckMessage(MSG_KEY_BLOCK_NESTED),
+            "23:15: " + getCheckMessage(MSG_KEY_BLOCK_NESTED),
         };
 
-        verifyWithInlineConfigParser(getPath("Example1.txt"), expected);
+        verifyWithInlineConfigParser(getPath("Example1.java"), expected);
     }
 
     @Test
     public void testExample2() throws Exception {
         final String[] expected = {
-
+            "19:5: " + getCheckMessage(MSG_KEY_BLOCK_NESTED),
         };
 
-        verifyWithInlineConfigParser(getPath("Example2.txt"), expected);
```

Comment:
Please remove such files at all from git.
We should have only java files.
Please use java files in template xml file.

Reply:
sure. @romani